### PR TITLE
Okteto manifest update for litmus-portal.

### PR DESCRIPTION
**Pull request by ishangupta-ds**
## Proposed changes

Okteto manifest update for litmus-portal.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- None.

## Special notes for your reviewer:
Bug-fix / Update

### CODE REVIEW

File: litmus-portal/namespaced-K8s-template.yml [hunk 1]
```diff
@@ -153,9 +153,7 @@ spec:
                   name: litmus-portal-admin-config
                   key: JWTSecret
             - name: LITMUS_PORTAL_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
+              value: ${LITMUS_PORTAL_NAMESPACE}
```
Comment (gdsoumya) [inline litmus-portal/namespaced-K8s-template.yml:156]: How will this value be parsed?

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: litmus-portal/namespaced-K8s-template.yml (hunk 1 above)
Comment (mdinc-ishangupta) [reply litmus-portal/namespaced-K8s-template.yml:156]: It is being taken as user input all over the manifest.. @gdsoumya cc: @rajdas98

Comment (ishangupta-ds) [reply litmus-portal/namespaced-K8s-template.yml:156]: it is taken as user input  all over the manifest @gdsoumya cc: @rajdas98

**Review by rajdas98 — APPROVED** [review]

File: litmus-portal/namespaced-K8s-template.yml (hunk 1 above)
Comment (gdsoumya) [reply litmus-portal/namespaced-K8s-template.yml:156]: Did not get you? How is this value being recieved? This manifest will be applied directly by the user using `kubectl apply` right?

**Pull request merged**